src/main/java/org/folio/circulation/domain/LoanAndRelatedRecords.java

```diff
@@ -9,69 +8,57 @@
   private final Loan loan;
   private final RequestQueue requestQueue;
   private final LoanPolicy loanPolicy;
-  private final AdjacentOpeningDays adjacentOpeningDays;
```

Comment:
Why is this being removed with this PR? I think code clean up is great, but, when it is unrelated to the story, I think it should be handled via a separate PR/story.

Reply:
Agree, reverted these changes.